lib/fs.js

```diff
@@ -101,6 +101,10 @@ function nullCheck(path, callback) {
   return true;
 }
 
+function isFd(path) {
+  return (path >>> 0) === path && path >= 0;
```

Comment:
To clarify, if the user passes `readFile('0xff')`, would that be interpreted as a file name or file descriptor?